index.html

```diff
@@ -168,16 +185,23 @@ <h1 class="card-heading">Professional Art Printing Data</h1>
       </div>
     </section>
     <section class="about-section">
```

Comment:
- [ ] Your 'About me' section has a lot of blank space at the top and bottom making it a little bit different from the Figma template. Kindly try to remove that blank space. Your design is really good!
Your design:
![image](https://user-images.githubusercontent.com/61919617/178038418-fb748991-0f17-419f-a52e-308d50486928.png)
Figma design:
![image](https://user-images.githubusercontent.com/61919617/178038696-05ad2752-03d9-4474-962f-e05c1f02837a.png)